tests/v2/validation/provisioning/clustertemplates/provisioning_cluster_template_test.go

```diff
@@ -0,0 +1,335 @@
+		etcdNodeCount, _ := etcdsnapshot.MatchNodeToAnyEtcdRole(client, clusterObject.ID)
+
+		_, err = provisioning.VerifySnapshots(client, clusterObject.Name, etcdNodeCount+len(existingSnapshots), true)
+		require.NoError(c.T(), err)
+	}
+
+	return clusterObject
+}
+
+func (c *ClusterTemplateProvisioningTestSuite) TestProvisioningRKE1ClusterWithClusterTemplate() {
+	log.Info("Create an rke template and creating a downstream node driver with the rke template.")
+
+	templateRevisionConfig := new(clustertemplates.ClusterTemplateRevisionConfig)
+	templateRevisionConfig.NetworkPlugin = c.provisioningConfig.CNIs[0]
+	templateRevisionConfig.RKE1KubernetesVersion = c.provisioningConfig.RKE1KubernetesVersions[0]
+
+	if c.provisioningConfig.ETCDRKE1 != nil {
+		templateRevisionConfig.BackupConfig = c.provisioningConfig.ETCDRKE1.BackupConfig
+	}
+
+	clusterConfig := c.createRke1ClusterTemplateAndTemplateRevisions(*templateRevisionConfig, nil, nil)
+
+	c.testCreateRke1ClusterWithTemplateAndVerify(c.client, clusterConfig)
+}
+
+func (c *ClusterTemplateProvisioningTestSuite) TestEnforceClusterTemplateProvisioningRKE1Cluster() {
+	log.Info("Enforcing cluster template while provisioning rke1 clusters")
+
+	steveAdminClient := c.client.Steve
+
+	clusterEnforcement, err := steveAdminClient.SteveType(settings.ManagementSetting).ByID(clusterEnforcementSetting)
+	require.NoError(c.T(), err)
+
+	_, err = settings.UpdateGlobalSettings(steveAdminClient, clusterEnforcement, enabledClusterEnforcementSetting)
+	require.NoError(c.T(), err)
+
+	verifySetting, err := steveAdminClient.SteveType(settings.ManagementSetting).ByID(clusterEnforcementSetting)
+	require.NoError(c.T(), err)
+	require.Equal(c.T(), verifySetting.JSONResp["value"], enabledClusterEnforcementSetting)
+
+	log.Info("Create a standard user and add them to the rke template.")
+	user, err := users.CreateUserWithRole(c.client, users.UserConfig(), "user")
+	require.NoError(c.T(), err)
+	standardClient, err := c.client.AsUser(user)
+	require.NoError(c.T(), err)
+
+	templateRevisionConfig := new(clustertemplates.ClusterTemplateRevisionConfig)
+	templateRevisionConfig.Name = namegenerator.AppendRandomString("rev1")
+	templateRevisionConfig.NetworkPlugin = c.provisioningConfig.CNIs[0]
+	templateRevisionConfig.RKE1KubernetesVersion = c.provisioningConfig.RKE1KubernetesVersions[0]
+	members := []mgmt.Member{
+		0: {AccessType: "owner",
+			UserPrincipalID: userPrincipalID + user.ID},
+	}
+
+	if c.provisioningConfig.ETCDRKE1 != nil {
+		templateRevisionConfig.BackupConfig = c.provisioningConfig.ETCDRKE1.BackupConfig
+	}
+
+	log.Info("Create a downstream cluster as the standard user.")
+	clusterConfig := c.createRke1ClusterTemplateAndTemplateRevisions(*templateRevisionConfig, nil, members)
+	c.testCreateRke1ClusterWithTemplateAndVerify(standardClient, clusterConfig)
+
+}
+
+func (c *ClusterTemplateProvisioningTestSuite) TestClusterTemplateWithQuestionsProvisioningRKE1Cluster() {
```

Comment:
ProvisioningRKE1 is already part of the suite name, and can be removed from the test name.